### PR TITLE
feat(sdk-coin-bsc): create bsc module

### PR DESCRIPTION
**Pull request by dulhanwaduge**
Ticket: BG-52150

## Changes
- Initialize the sdk-coin-bsc (Binance Smart Chain) module

### CODE REVIEW

**Review by wesgraham — APPROVED** [review]

File: modules/account-lib/package.json [hunk 1]
```diff
@@ -34,6 +34,7 @@
     "@bitgo/sdk-coin-algo": "^1.0.0",
     "@bitgo/sdk-coin-avaxc": "^1.0.0",
     "@bitgo/sdk-coin-avaxp": "^1.0.0",
+    "@bitgo/sdk-coin-bsc": "^1.0.0",
```
Comment (mmcshinsky-bitgo) [inline modules/account-lib/package.json:37]: Need to add to tsconfig as well.

**Review by mmcshinsky-bitgo — COMMENTED** [review]
LGTM. Just needs the tsconfig and you are good to go! :)

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Review by wesgraham — APPROVED** [review]

File: modules/sdk-coin-bsc/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,3 @@
+export * as BscLib from './lib';
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-coin-bsc/src/index.ts:1]: Undocumented, but we're going go export everything without the name. I need to change this in the sdk-coin:new script.

Comment (dulhanwaduge) [reply modules/sdk-coin-bsc/src/index.ts:1]: ah ok, will do right now.

File: modules/sdk-coin-bsc/tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "esModuleInterop": true,
+    "typeRoots": ["../../types", "./node_modules/@types", "../../node_modules/@types"]
+  },
+  "include": ["src/**/*", "test/**/*"],
+  "exclude": ["node_modules"],
+  "references": [
+    {
+      "path": "../abstract-eth"
+    },
+    {
```
Comment (mmcshinsky-bitgo) [inline modules/sdk-coin-bsc/tsconfig.json:16]: Needs `sdk-coin-eth`

**Review by mmcshinsky-bitgo — CHANGES_REQUESTED** [review]
Looks great! One minor thing and we're good to go!

**Review by mmcshinsky-bitgo — APPROVED** [review]

**Pull request merged**